timeforcoffee.xcodeproj/project.pbxproj

```diff
@@ -754,7 +757,7 @@
 					};
 					EC46C5271BB5C78A00F2A963 = {
 						CreatedOnToolsVersion = 7.0;
-						DevelopmentTeam = DN76797UYF;
+						DevelopmentTeam = 6EMM22S2YK;
 						SystemCapabilities = {
```

Comment:
Oh great ! I didn't notice this alternative method for getDestination. Sure
I ll do the changes and rebase against master. If you see anything else,
let me know.

2015-10-14 9:03 GMT+02:00 Christian Stocker <notifications@github.com>:

> In timeforcoffee.xcodeproj/project.pbxproj
> <https://github.com/timeforcoffee/timeforcoffee-ios/pull/4#discussion_r41960777>
> :
>
> > @@ -754,7 +757,7 @@
> >  					};
> >  					EC46C5271BB5C78A00F2A963 = {
> >  						CreatedOnToolsVersion = 7.0;
> > -						DevelopmentTeam = DN76797UYF;
> > +						DevelopmentTeam = 6EMM22S2YK;
> >  						SystemCapabilities = {
>
> can you please move the DevTeam back them and the PROVISIONING_PROFILE
> changes below as well
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/timeforcoffee/timeforcoffee-ios/pull/4/files#r41960777>
> .
>